dnora/grd/read.py

```diff
@@ -126,7 +126,7 @@ def _crop(ds):
             return ds.isel(lon=slice(2, -1), lat=slice(2, -1))
 
         import dask
-        with dask.config.set(**{'array.slicing.split_large_chunks': True}):
```

Comment:
also I now think that should be True here. Haven't properly benchmarked it though.

Reply:
Ok, I changed that flag to True.